Adafruit_LEDBackpack.cpp

```diff
@@ -633,24 +633,16 @@ void Adafruit_7segment::println(double n, int digits) {
 void Adafruit_7segment::print(double n, int digits) { printFloat(n, digits); }
 
 size_t Adafruit_7segment::write(char c) {
-
-  uint8_t r = 0;
-
-  if (c == '\n')
-    position = 0;
-  if (c == '\r')
+  if (c == '\n' || c == '\r')
     position = 0;
 
-  if ((c >= ' ') && (c <= 127)) {
-    writeDigitAscii(position, c);
-    r = 1;
-  }
```

Comment:
Why remove this check?

Reply:
Its purpose was to check whether the character is present in the font table. That functionality has been moved to `writeDigitAscii()` where the font table is accessed.